src/packages/user/user/workspace/user-workspace.context.ts

```diff
@@ -26,19 +26,31 @@ export class UmbUserWorkspaceContext
 	data = this.#data.asObservable();
 
 	async load(id: string) {
-		const { data } = await this.repository.requestById(id);
+		const { data, asObservable } = await this.repository.requestById(id);
 		if (data) {
 			this.setIsNew(false);
 			this.#data.update(data);
 		}
+
+		this.observe(asObservable(), (user) => this.onUserStoreChanges(user));
```

Comment:
note about missing ControllerAlias, to make observe replace it self if this method is called again.